packages/asset-swapper/src/types.ts

```diff
@@ -155,6 +156,7 @@ export interface SwapQuoteBase {
     bestCaseQuoteInfo: SwapQuoteInfo;
     worstCaseQuoteInfo: SwapQuoteInfo;
     sourceBreakdown: SwapQuoteOrdersBreakdown;
+    quoteReport?: QuoteReport;
```

Comment:
Made this optional because I didn't know if it should return for the `getBatchMarket`... requests

Reply:
wouldn't there be one per swap quote? `getBatch` is just performing N many swap quotes at once.

Reply:
i didn't feel as though it was worth going into the `_calculateBatchBuySwapQuoteAsync` functions and returning `QuoteReport`s because I don't think there would actually be a use for it.  happy to revisit if you think it's important or could be useful though